### PR TITLE
Add Welds' RIG

### PR DESCRIPTION
**Pull request by KittenColony**
## About the PR
Adds a personal item for Welds, specifically the RIG they have had in their description for _months_

## Why / Balance
Welds is can walk!

## Technical details
Sprites and YAML

## Media
image
imageimage

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

nuh uh

**Changelog**

:cl:

* add: Welds' RIG

### CODE REVIEW

**Comment by TsjipTsjip** [comment]
At a glance on my phone, this looks okay. Will test properly this weekend.

**Comment by Simyon264** [comment]
Looks all good, will have to test ingame once I'm home.

**Review by Simyon264 — APPROVED** [review]
image

It works ingame. One small nitpick which is not that important: The footstep sounds have a weird echo that you would expect from maints, not in a hallway. But that is not very noticable.

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]
GM approval, but one comment that needs to be addressed before this can be merged. I was not given write access to the branch to resolve this myself.

File: Resources/Prototypes/_Umbra/Entities/Clothing/Shoes/misc.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingShoesMiscWeldsWalker
+  name: Welds' R.I.G
+  description: Made from an ATMOS hardsuit and a pair of Magboots. You don't know how, but it has gyroscoptic capabilities. No wonder They're still here.
```
Comment (TsjipTsjip) [inline Resources/Prototypes/_Umbra/Entities/Clothing/Shoes/misc.yml:5]: What's going on with the capital T in the last sentence? Is this intentional?

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]
Guh! I didn't click save on the comment!

File: Resources/Prototypes/_Umbra/Entities/Clothing/Shoes/misc.yml (hunk 1 above)
Comment (KittenColony) [reply Resources/Prototypes/_Umbra/Entities/Clothing/Shoes/misc.yml:5]: Welds is non-binary, same logic behind H(e) S(he), dunno if its correct though

Comment (TsjipTsjip) [reply Resources/Prototypes/_Umbra/Entities/Clothing/Shoes/misc.yml:5]: Fair, but that wasn't the issue I wanted to point out. Personal pronouns generally shouldn't be capitalized, I was wondering if there was a reason for that here.

Comment (KittenColony) [reply Resources/Prototypes/_Umbra/Entities/Clothing/Shoes/misc.yml:5]: in that case this is incorrect then

File: Resources/Prototypes/_Umbra/Loadouts/Miscellaneous/personal_items.yml [hunk 2]
```diff
@@ -124,6 +124,20 @@
...
+  equipment: ClothingShoesMiscWeldsWalker
+  effects:
+  - !type:PersonalItemLoadoutEffect
+    character: Welds-The-Pipes
+
+- type: startingGear
+  id: ClothingShoesMiscWeldsWalker
+  storage:
+    back:
+    - ClothingShoesMiscWeldsWalker
```
Comment (arimah) [inline Resources/Prototypes/_Umbra/Loadouts/Miscellaneous/personal_items.yml:139]: In addition to this (which is all correct), the item must _also_ be added to every applicable `loadoutGroup` defined in `Resources/Prototypes/_Umbra/Loadouts/loadout_groups.yml`. Each job has a set of loadout groups, each loadout group has a set of loadouts, each loadout has a starting gear (equipment), each starting gear assigns items... Yes, the loadouts system is a mess.

**Review by arimah — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_Umbra/Loadouts/Miscellaneous/personal_items.yml (hunk 2 above)
Comment (TsjipTsjip) [reply Resources/Prototypes/_Umbra/Loadouts/Miscellaneous/personal_items.yml:139]: Awww man.... I'm not even done screaming at dotnet yet and you're giving me this?

This is my bad. I'll resolve this in a bit.

Comment (TsjipTsjip) [reply Resources/Prototypes/_Umbra/Loadouts/Miscellaneous/personal_items.yml:139]: Resolved by 470e983 .

**Comment by TsjipTsjip** [comment]
Still needs testing in-game and then this is good to ship.

**Pull request merged**